pkg/search/proxy/framework/runtime/framework.go

```diff
@@ -0,0 +1,51 @@
+package runtime
```

Comment:
has a better name? runtime not clear enough for the real usage?

Reply:
In alignment with `pkg/scheduler/framework/runtime`